tests/IntegrationTests/WebAppUiTests/TestingWebAppCallsApiCallsGraphLocally.cs

```diff
@@ -1,139 +1,145 @@
-                await Assertions.Expect(page.GetByRole(AriaRole.Cell, new() { Name = TodoTitle1 })).ToBeVisibleAsync();
-                _output.WriteLine("Web app create new todo flow successful.");
-
-                // Edit todo item
-                _output.WriteLine("Starting web app edit todo flow.");
-                await page.GetByRole(AriaRole.Link, new() { Name = "Edit" }).ClickAsync();
-                await UiTestHelpers.FillEntryBox(TitleEntryBox, TodoTitle2);
-                await Assertions.Expect(page.GetByRole(AriaRole.Cell, new() { Name = TodoTitle2 })).ToBeVisibleAsync();
-                _output.WriteLine("Web app edit todo flow successful.");
-
-                // Delete todo item
-                _output.WriteLine("Starting web app delete todo flow.");
-                await page.GetByRole(AriaRole.Link, new() { Name = "Delete" }).ClickAsync();
-                await page.GetByRole(AriaRole.Button, new() { Name = "Delete" }).ClickAsync();
-                await Assertions.Expect(page.GetByRole(AriaRole.Cell, new() { Name = TodoTitle2 })).Not.ToBeVisibleAsync();
-                _output.WriteLine("Web app delete todo flow successful.");
-            }
-            catch (System.Exception ex)
-            {
-                Assert.Fail($"the UI automation failed: {ex} output: {ex.Message}.");
-            }
-            finally
-            {
-                // Add the following to make sure all processes and their children are stopped 
-                Queue<Process> processes = new Queue<Process>();
-                processes.Enqueue(serviceProcess!);
-                processes.Enqueue(clientProcess!);
-                processes.Enqueue(grpcProcess!);
-                UiTestHelpers.KillProcessTrees(processes);
-
-                // Stop tracing and export it into a zip archive.
-                string path = UiTestHelpers.GetTracePath(_uiTestAssemblyLocation, TraceFileName);
-                await context.Tracing.StopAsync(new() { Path = path });
-                _output.WriteLine($"Trace data for {TraceFileName} recorded to {path}.");
-
-                // Close the browser and stop Playwright.
-                await browser.CloseAsync();
-                playwright.Dispose();
-            }
-        }
-    }
-#endif // !FROM_GITHUB_ACTION
-}
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.Versioning;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Identity.Lab.Api;
+using Microsoft.Playwright;
+using Xunit;
+using Xunit.Abstractions;
+using Process = System.Diagnostics.Process;
+
+namespace WebAppUiTests
+#if !FROM_GITHUB_ACTION
+{
+    [CollectionDefinition(nameof(TestingWebAppCallsApiCallsGraphLocally), DisableParallelization = true)] // since this changes environment variables we'd prefer it not run at the same time as other tests
```

Comment:
Nit: stay consistent with putting comments on their own line, directly above the one to which they refer

Reply:
altrered